app/pb_kits/playbook/pb_text_input/_text_input.html.erb

```diff
@@ -1,12 +1,12 @@
 <%= content_tag(:div,
     class: object.classname) do %>
   <% if object.label.present? %>
-    <%= pb_rails("caption", props: {text: object.label}) %>
+    <%= pb_rails("caption", props: {text: object.label, dark: object.dark}) %>
```

Comment:
I like that is is automatically passing the dark option down into the caption kit. Any thoughts of adding support for this to the form kit, so we could specify the form as being a dark mode variant, and having this option to automatically pushed down into the text input kit and button kit?

Reply:
Love it @terryfinn . We will start a PR for that